apps/src/lib/client/rpc.rs

```diff
@@ -923,14 +931,19 @@ pub async fn query_voting_power(ctx: Context, args: args::QueryVotingPower) {
                         );
                     }
                     println!(
-                        "Validator {} is {}, voting power: {}",
+                        "Validator {} is {}, bonded stake: {}, voting power: \
+                         {}",
                         validator.encode(),
                         if is_active { "active" } else { "inactive" },
-                        voting_power
+                        bonded_stake,
+                        (Decimal::from(bonded_stake)
+                            * pos_params.tm_votes_per_token)
```

Comment:
I think perhaps for most part, we could use the `bonded_stake` value as is, because `tm_votes_per_token` will only be used in tendermint validator set

Reply:
Here I included the voting power in the output because `namada_tests::e2e::helpers::find_voting_power` works by looking for this string in std::out. I guess we could maybe change `find_voting_power` in favor of `find_bonded_stake` or something, but I figured this would be useful from the client-side.

Reply:
I see - I think we could replace it with `find_bonded_stake` as in majority of places we'll be using that, `client voting-power` command might only be confusing